tests/network/vmi_multus.go

```diff
@@ -566,6 +568,75 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 				Expect(testErr.ErrStatus.Reason).To(BeEquivalentTo("Invalid"))
 			})
 		})
+
+		Context("MAC Spoof check with Linux bridge CNI plugin interface and custom MAC address.", func() {
+			It("Should allow outbound communication from VM under test - only if original MAC address is unchanged", func() {
+				const (
+					bridgeInterfaceName = "linux-bridge"
+					initialMacAddress   = "50:00:00:00:90:0e"
+					spoofedMacAddress   = "50:00:00:00:80:0a"
+				)
+
+				const (
+					vmUnderTestIPAddress = "10.2.1.1"
+					dummyVMIPAddress     = "10.2.1.2"
+					bridgeSubnetMask     = "/24"
+				)
+
+				linuxBridgeInterfaceWithMACSpoofCheck := v1.Interface{
+					Name: bridgeInterfaceName,
+					InterfaceBindingMethod: v1.InterfaceBindingMethod{
+						Bridge: &v1.InterfaceBridge{},
+					},
+				}
+
+				linuxBridgeWithMACSpoofCheckNetwork := v1.Network{
+					Name: bridgeInterfaceName,
+					NetworkSource: v1.NetworkSource{
+						Multus: &v1.MultusNetwork{
+							NetworkName: linuxBridgeWithMACSpoofCheckNetworkName,
+						},
+					},
+				}
+
+				By("Creating a VM with custom MAC address on its Linux bridge CNI interface.")
+				linuxBridgeInterfaceWithCustomMac := linuxBridgeInterfaceWithMACSpoofCheck
+				linuxBridgeInterfaceWithCustomMac.MacAddress = initialMacAddress
+				vmiUnderTest := libvmi.NewTestToolingFedora(
+					libvmi.WithInterface(linuxBridgeInterfaceWithCustomMac),
+					libvmi.WithNetwork(&linuxBridgeWithMACSpoofCheckNetwork),
+					libvmi.WithCloudInitNoCloudNetworkData(cloudInitNetworkDataWithStaticIPsByMac(linuxBridgeInterfaceWithCustomMac.Name, initialMacAddress, vmUnderTestIPAddress+bridgeSubnetMask), false))
+				vmiUnderTest = tests.CreateVmiOnNode(vmiUnderTest, nodes.Items[0].Name)
+
+				By("Creating a dummy VM with Linux bridge CNI network interface and default MAC address.")
+				dummyVmi := libvmi.NewTestToolingFedora(
+					libvmi.WithInterface(linuxBridgeInterfaceWithMACSpoofCheck),
+					libvmi.WithNetwork(&linuxBridgeWithMACSpoofCheckNetwork),
+					libvmi.WithCloudInitNoCloudNetworkData(cloudInitNetworkDataWithStaticIPsByDevice("eth0", dummyVMIPAddress+bridgeSubnetMask), false))
+				dummyVmi = tests.CreateVmiOnNode(dummyVmi, nodes.Items[0].Name)
+
+				By("Waiting for VM under test to be ready.")
+				vmiUnderTest = tests.WaitUntilVMIReady(vmiUnderTest, console.LoginToFedora)
+
+				By("Waiting for dummy VM to be ready.")
+				tests.WaitUntilVMIReady(dummyVmi, console.LoginToFedora)
+
+				By("Pinging dummy VM from VM under test.")
+				Expect(libnet.PingFromVMConsole(vmiUnderTest, dummyVMIPAddress)).To(Succeed())
```

Comment:
You can add an assertion message like: "Connectivity check with original MAC failed".
And something similar after the MAC change.